src/vs/workbench/contrib/search/browser/searchActionsTopBar.ts

```diff
@@ -252,7 +252,7 @@ async function expandAll(accessor: ServicesAccessor) {
 			if (searchView.model.hasAIResults) {
 				await forcedExpandRecursively(viewer, undefined);
 			} else {
-				await forcedExpandRecursively(viewer, searchView.model.searchResult.plainTextSearchResult);
+				await forcedExpandRecursively(viewer, undefined);
```

Comment:
All these statements now do the same thing, could we just remove all the nesting and execute `forcedExpandRecursively` for all cases?